File: weekly/336/q1.py
class Solution:
    def vowelStrings(self, words: List[str], left: int, right: int) -> int:
        ans = 0
        VOWELS = 'aeiou'
        for i in range(left, right + 1):
            if words[i][0] in VOWELS and words[i][-1] in VOWELS:
                ans += 1
        return ans

File: biweekly/107/q1.cc
class Solution {
public:
    int maximumNumberOfStringPairs(vector<string>& words) {
        set<string> s;
        int ans = 0;
        for (auto w : words) {
            if (s.count(w)) {
                ans++;
            }
            reverse(w.begin(), w.end());
            s.insert(w);
        }
        return ans;
    }
};

